src/profile_manager/models.py

```diff
@@ -448,7 +479,7 @@ def smart_list(value, delimiter=",", func=None):
     model signal to ensure that you always get a list back from the field.
 
     """
-    if value in ["", u"", "[]", u"[]", u"[ ]", None]:
+    if value in ["", "", "[]", "[]", "[ ]", None]:
```

Comment:
Hmm... this looks sus.  Now it's just repeating values.  Obviously `u""` isn't tested, or this change would fail, but unless you have a specific reason for changing this, please leave as is.

Reply:
I think this is from one of the pre-commit hooks and I think the `u` is not relevant anymore as of python 3

Reply:
You're right! In that case, we can remove the repeated list items.